## `nixpkgs-review` result for [#479875](https://github.com/NixOS/nixpkgs/pull/479875)

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 479875`
Commit: [`1aaaa5e2c5828acfa0505ae8c9d3a1c113440db1`](https://github.com/NixOS/nixpkgs/commit/1aaaa5e2c5828acfa0505ae8c9d3a1c113440db1) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/1aaaa5e2c5828acfa0505ae8c9d3a1c113440db1..pull/479875/head))
Merge: [`fd1373bfc4869cb19af90d821a97b85585f2cf14`](https://github.com/NixOS/nixpkgs/commit/fd1373bfc4869cb19af90d821a97b85585f2cf14)

Logs: https://github.com/SkohTV/nixpkgs-review-gha/actions/runs/20977180850


---
### `x86_64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python313Packages.hikari-crescent</li>
    <li>python313Packages.hikari-crescent.dist</li>
    <li>python314Packages.hikari-crescent</li>
    <li>python314Packages.hikari-crescent.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python313Packages.hikari</li>
    <li>python313Packages.hikari.dist</li>
    <li>python314Packages.hikari</li>
    <li>python314Packages.hikari.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python313Packages.hikari-crescent</summary>
<pre>/nix/store/pvp1hn0v9g5rg5jna30k7y8k2mq5ssdc-python3.13-hikari-2.5.0/lib/python3.13/site-packages/hikari/impl/gateway_bot.py:346: in __init__
    self._token_id = applications.get_token_id(self._token)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/nix/store/pvp1hn0v9g5rg5jna30k7y8k2mq5ssdc-python3.13-hikari-2.5.0/lib/python3.13/site-packages/hikari/applications.py:1015: in get_token_id
    raise ValueError(msg) from exc
E   ValueError: Unexpected token format
------------------------------- Captured stdout --------------------------------

  oooo         o8o  oooo                            o8o       光 2.5.0 [2a605db9]
  `888         `&quot;&#x27;  `888                            `&quot;&#x27;       © 2021-present, davfsa - MIT license
   888 .oo.   oooo   888  oooo   .oooo.   oooo d8b oooo       interpreter:   CPython 3.13.11
   888P&quot;Y88b  `888   888 .8P&#x27;   `P  )88b  `888&quot;&quot;8P `888       running on:    x86_64 Linux 6.11.0-1018-azure
   888   888   888   888888.     .oP&quot;888   888      888       installed at:  /nix/store/pvp1hn0v9g5rg5jna30k7y8k2mq5ssdc-python3.13-hikari-2.5.0/lib/python3.13/site-packages/hikari
   888   888   888   888 `88b.  d8(  888   888      888       documentation: https://docs.hikari-py.dev/en/2.5.0
  o888o o888o o888o o888o o888o `Y888&quot;&quot;8o d888b    o888o      support:       https://discord.gg/Jx4cNGG

=========================== short test summary info ============================
ERROR tests/test_bot/test_bot.py - ValueError: Unexpected token format
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 11 deselected, 1 error in 0.84s ========================</pre>
</details>
<details>
<summary>python314Packages.hikari-crescent</summary>
<pre>/nix/store/kmxxw0c3zbxw5c67f2p0jhi4mqg93agv-python3.14-hikari-2.5.0/lib/python3.14/site-packages/hikari/impl/gateway_bot.py:346: in __init__
    self._token_id = applications.get_token_id(self._token)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/nix/store/kmxxw0c3zbxw5c67f2p0jhi4mqg93agv-python3.14-hikari-2.5.0/lib/python3.14/site-packages/hikari/applications.py:1015: in get_token_id
    raise ValueError(msg) from exc
E   ValueError: Unexpected token format
------------------------------- Captured stdout --------------------------------

  oooo         o8o  oooo                            o8o       光 2.5.0 [2a605db9]
  `888         `&quot;&#x27;  `888                            `&quot;&#x27;       © 2021-present, davfsa - MIT license
   888 .oo.   oooo   888  oooo   .oooo.   oooo d8b oooo       interpreter:   CPython 3.14.2
   888P&quot;Y88b  `888   888 .8P&#x27;   `P  )88b  `888&quot;&quot;8P `888       running on:    x86_64 Linux 6.11.0-1018-azure
   888   888   888   888888.     .oP&quot;888   888      888       installed at:  /nix/store/kmxxw0c3zbxw5c67f2p0jhi4mqg93agv-python3.14-hikari-2.5.0/lib/python3.14/site-packages/hikari
   888   888   888   888 `88b.  d8(  888   888      888       documentation: https://docs.hikari-py.dev/en/2.5.0
  o888o o888o o888o o888o o888o `Y888&quot;&quot;8o d888b    o888o      support:       https://discord.gg/Jx4cNGG

=========================== short test summary info ============================
ERROR tests/test_bot/test_bot.py - ValueError: Unexpected token format
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 11 deselected, 1 error in 0.87s ========================</pre>
</details>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python313Packages.hikari-crescent</li>
    <li>python313Packages.hikari-crescent.dist</li>
    <li>python314Packages.hikari-crescent</li>
    <li>python314Packages.hikari-crescent.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python313Packages.hikari</li>
    <li>python313Packages.hikari.dist</li>
    <li>python314Packages.hikari</li>
    <li>python314Packages.hikari.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>python313Packages.hikari-crescent</summary>
<pre>/nix/store/nap873jjn8y5yvxaarnpggaapk2fjsbp-python3.13-hikari-2.5.0/lib/python3.13/site-packages/hikari/impl/gateway_bot.py:346: in __init__
    self._token_id = applications.get_token_id(self._token)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/nix/store/nap873jjn8y5yvxaarnpggaapk2fjsbp-python3.13-hikari-2.5.0/lib/python3.13/site-packages/hikari/applications.py:1015: in get_token_id
    raise ValueError(msg) from exc
E   ValueError: Unexpected token format
------------------------------- Captured stdout --------------------------------

  oooo         o8o  oooo                            o8o       光 2.5.0 [2a605db9]
  `888         `&quot;&#x27;  `888                            `&quot;&#x27;       © 2021-present, davfsa - MIT license
   888 .oo.   oooo   888  oooo   .oooo.   oooo d8b oooo       interpreter:   CPython 3.13.11
   888P&quot;Y88b  `888   888 .8P&#x27;   `P  )88b  `888&quot;&quot;8P `888       running on:    aarch64 Linux 6.14.0-1014-azure
   888   888   888   888888.     .oP&quot;888   888      888       installed at:  /nix/store/nap873jjn8y5yvxaarnpggaapk2fjsbp-python3.13-hikari-2.5.0/lib/python3.13/site-packages/hikari
   888   888   888   888 `88b.  d8(  888   888      888       documentation: https://docs.hikari-py.dev/en/2.5.0
  o888o o888o o888o o888o o888o `Y888&quot;&quot;8o d888b    o888o      support:       https://discord.gg/Jx4cNGG

=========================== short test summary info ============================
ERROR tests/test_bot/test_bot.py - ValueError: Unexpected token format
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 11 deselected, 1 error in 0.74s ========================</pre>
</details>
<details>
<summary>python314Packages.hikari-crescent</summary>
<pre>/nix/store/vq198mjc6s4p7wjssk9llg83d1sp4sz9-python3.14-hikari-2.5.0/lib/python3.14/site-packages/hikari/impl/gateway_bot.py:346: in __init__
    self._token_id = applications.get_token_id(self._token)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/nix/store/vq198mjc6s4p7wjssk9llg83d1sp4sz9-python3.14-hikari-2.5.0/lib/python3.14/site-packages/hikari/applications.py:1015: in get_token_id
    raise ValueError(msg) from exc
E   ValueError: Unexpected token format
------------------------------- Captured stdout --------------------------------

  oooo         o8o  oooo                            o8o       光 2.5.0 [2a605db9]
  `888         `&quot;&#x27;  `888                            `&quot;&#x27;       © 2021-present, davfsa - MIT license
   888 .oo.   oooo   888  oooo   .oooo.   oooo d8b oooo       interpreter:   CPython 3.14.2
   888P&quot;Y88b  `888   888 .8P&#x27;   `P  )88b  `888&quot;&quot;8P `888       running on:    aarch64 Linux 6.14.0-1014-azure
   888   888   888   888888.     .oP&quot;888   888      888       installed at:  /nix/store/vq198mjc6s4p7wjssk9llg83d1sp4sz9-python3.14-hikari-2.5.0/lib/python3.14/site-packages/hikari
   888   888   888   888 `88b.  d8(  888   888      888       documentation: https://docs.hikari-py.dev/en/2.5.0
  o888o o888o o888o o888o o888o `Y888&quot;&quot;8o d888b    o888o      support:       https://discord.gg/Jx4cNGG

=========================== short test summary info ============================
ERROR tests/test_bot/test_bot.py - ValueError: Unexpected token format
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 11 deselected, 1 error in 0.76s ========================</pre>
</details>
</details>

---
### `x86_64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python313Packages.hikari-crescent</li>
    <li>python313Packages.hikari-crescent.dist</li>
    <li>python314Packages.hikari-crescent</li>
    <li>python314Packages.hikari-crescent.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python313Packages.hikari</li>
    <li>python313Packages.hikari.dist</li>
    <li>python314Packages.hikari</li>
    <li>python314Packages.hikari.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>python313Packages.hikari-crescent</summary>
<pre>/nix/store/bdg9mqyalfn1xkwvy3h64wsczsi4q8i9-python3.13-hikari-2.5.0/lib/python3.13/site-packages/hikari/impl/gateway_bot.py:346: in __init__
    self._token_id = applications.get_token_id(self._token)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/nix/store/bdg9mqyalfn1xkwvy3h64wsczsi4q8i9-python3.13-hikari-2.5.0/lib/python3.13/site-packages/hikari/applications.py:1015: in get_token_id
    raise ValueError(msg) from exc
E   ValueError: Unexpected token format
------------------------------- Captured stdout --------------------------------

  oooo         o8o  oooo                            o8o       光 2.5.0 [2a605db9]
  `888         `&quot;&#x27;  `888                            `&quot;&#x27;       © 2021-present, davfsa - MIT license
   888 .oo.   oooo   888  oooo   .oooo.   oooo d8b oooo       interpreter:   CPython 3.13.11
   888P&quot;Y88b  `888   888 .8P&#x27;   `P  )88b  `888&quot;&quot;8P `888       running on:    x86_64 Darwin 24.6.0
   888   888   888   888888.     .oP&quot;888   888      888       installed at:  /nix/store/bdg9mqyalfn1xkwvy3h64wsczsi4q8i9-python3.13-hikari-2.5.0/lib/python3.13/site-packages/hikari
   888   888   888   888 `88b.  d8(  888   888      888       documentation: https://docs.hikari-py.dev/en/2.5.0
  o888o o888o o888o o888o o888o `Y888&quot;&quot;8o d888b    o888o      support:       https://discord.gg/Jx4cNGG

=========================== short test summary info ============================
ERROR tests/test_bot/test_bot.py - ValueError: Unexpected token format
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 11 deselected, 1 error in 1.45s ========================</pre>
</details>
<details>
<summary>python314Packages.hikari-crescent</summary>
<pre>/nix/store/gl6qfsa2z461q8ymgprz5f28850jz3xq-python3.14-hikari-2.5.0/lib/python3.14/site-packages/hikari/impl/gateway_bot.py:346: in __init__
    self._token_id = applications.get_token_id(self._token)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/nix/store/gl6qfsa2z461q8ymgprz5f28850jz3xq-python3.14-hikari-2.5.0/lib/python3.14/site-packages/hikari/applications.py:1015: in get_token_id
    raise ValueError(msg) from exc
E   ValueError: Unexpected token format
------------------------------- Captured stdout --------------------------------

  oooo         o8o  oooo                            o8o       光 2.5.0 [2a605db9]
  `888         `&quot;&#x27;  `888                            `&quot;&#x27;       © 2021-present, davfsa - MIT license
   888 .oo.   oooo   888  oooo   .oooo.   oooo d8b oooo       interpreter:   CPython 3.14.2
   888P&quot;Y88b  `888   888 .8P&#x27;   `P  )88b  `888&quot;&quot;8P `888       running on:    x86_64 Darwin 24.6.0
   888   888   888   888888.     .oP&quot;888   888      888       installed at:  /nix/store/gl6qfsa2z461q8ymgprz5f28850jz3xq-python3.14-hikari-2.5.0/lib/python3.14/site-packages/hikari
   888   888   888   888 `88b.  d8(  888   888      888       documentation: https://docs.hikari-py.dev/en/2.5.0
  o888o o888o o888o o888o o888o `Y888&quot;&quot;8o d888b    o888o      support:       https://discord.gg/Jx4cNGG

=========================== short test summary info ============================
ERROR tests/test_bot/test_bot.py - ValueError: Unexpected token format
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 11 deselected, 1 error in 1.46s ========================</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python313Packages.hikari-crescent</li>
    <li>python313Packages.hikari-crescent.dist</li>
    <li>python314Packages.hikari-crescent</li>
    <li>python314Packages.hikari-crescent.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python313Packages.hikari</li>
    <li>python313Packages.hikari.dist</li>
    <li>python314Packages.hikari</li>
    <li>python314Packages.hikari.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python313Packages.hikari-crescent</summary>
<pre>/nix/store/grwy879gwf7w89h641yj7ml86bz6asf6-python3.13-hikari-2.5.0/lib/python3.13/site-packages/hikari/impl/gateway_bot.py:346: in __init__
    self._token_id = applications.get_token_id(self._token)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/nix/store/grwy879gwf7w89h641yj7ml86bz6asf6-python3.13-hikari-2.5.0/lib/python3.13/site-packages/hikari/applications.py:1015: in get_token_id
    raise ValueError(msg) from exc
E   ValueError: Unexpected token format
------------------------------- Captured stdout --------------------------------

  oooo         o8o  oooo                            o8o       光 2.5.0 [2a605db9]
  `888         `&quot;&#x27;  `888                            `&quot;&#x27;       © 2021-present, davfsa - MIT license
   888 .oo.   oooo   888  oooo   .oooo.   oooo d8b oooo       interpreter:   CPython 3.13.11
   888P&quot;Y88b  `888   888 .8P&#x27;   `P  )88b  `888&quot;&quot;8P `888       running on:    arm64 Darwin 24.6.0
   888   888   888   888888.     .oP&quot;888   888      888       installed at:  /nix/store/grwy879gwf7w89h641yj7ml86bz6asf6-python3.13-hikari-2.5.0/lib/python3.13/site-packages/hikari
   888   888   888   888 `88b.  d8(  888   888      888       documentation: https://docs.hikari-py.dev/en/2.5.0
  o888o o888o o888o o888o o888o `Y888&quot;&quot;8o d888b    o888o      support:       https://discord.gg/Jx4cNGG

=========================== short test summary info ============================
ERROR tests/test_bot/test_bot.py - ValueError: Unexpected token format
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 11 deselected, 1 error in 0.57s ========================</pre>
</details>
<details>
<summary>python314Packages.hikari-crescent</summary>
<pre>/nix/store/6403494zm4jcrccka8n2jfbjhwyg8v3m-python3.14-hikari-2.5.0/lib/python3.14/site-packages/hikari/impl/gateway_bot.py:346: in __init__
    self._token_id = applications.get_token_id(self._token)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/nix/store/6403494zm4jcrccka8n2jfbjhwyg8v3m-python3.14-hikari-2.5.0/lib/python3.14/site-packages/hikari/applications.py:1015: in get_token_id
    raise ValueError(msg) from exc
E   ValueError: Unexpected token format
------------------------------- Captured stdout --------------------------------

  oooo         o8o  oooo                            o8o       光 2.5.0 [2a605db9]
  `888         `&quot;&#x27;  `888                            `&quot;&#x27;       © 2021-present, davfsa - MIT license
   888 .oo.   oooo   888  oooo   .oooo.   oooo d8b oooo       interpreter:   CPython 3.14.2
   888P&quot;Y88b  `888   888 .8P&#x27;   `P  )88b  `888&quot;&quot;8P `888       running on:    arm64 Darwin 24.6.0
   888   888   888   888888.     .oP&quot;888   888      888       installed at:  /nix/store/6403494zm4jcrccka8n2jfbjhwyg8v3m-python3.14-hikari-2.5.0/lib/python3.14/site-packages/hikari
   888   888   888   888 `88b.  d8(  888   888      888       documentation: https://docs.hikari-py.dev/en/2.5.0
  o888o o888o o888o o888o o888o `Y888&quot;&quot;8o d888b    o888o      support:       https://discord.gg/Jx4cNGG

=========================== short test summary info ============================
ERROR tests/test_bot/test_bot.py - ValueError: Unexpected token format
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================= 11 deselected, 1 error in 0.58s ========================</pre>
</details>
</details>

--- a/pkgs/development/python-modules/hikari-crescent/default.nix
+++ b/pkgs/development/python-modules/hikari-crescent/default.nix
@@ -43,6 +43,8 @@ buildPythonPackage (finalAttrs: {
 
   disabledTests = [ "test_handle_resp" ];
 
+  disabledTestPaths = [ "tests/test_bot/test_bot.py" ];
+
   meta = {
     description = "Command handler for Hikari that keeps your project neat and tidy";
     license = lib.licenses.mit;

